scripts/SANS/sans/algorithm_detail/move_workspaces.py

```diff
@@ -538,7 +555,9 @@ class SANSMoveLARMOROldStyle(SANSMove):
     def __init__(self):
         super(SANSMoveLARMOROldStyle, self).__init__()
 
-    def do_move_initial(self, move_info, workspace, coordinates, component):
+    def do_move_initial(self, move_info, workspace, coordinates, component, is_transmission_workspace):
+        _is_transmission_workspace = is_transmission_workspace  # noqa
```

Comment:
The methods is defined on an abstract base class and that particular field is used at the moment only by `LOQ` in `SANSMoveLOQ`.